generator/mavschema.xsd

```diff
@@ -178,6 +178,7 @@
 <xs:simpleType name="factor">
   <xs:restriction base="xs:string">
     <xs:enumeration value="1E-2"/>        <!-- actual value = stated value / 100 -->
+    <xs:enumeration value="360/255"/>     <!-- actual value = stated value * 360/255, as used for GPS_STATUS.satellite_azimuth -->
```

Comment:
@peterbarker How about
```suggestion
    <xs:enumeration value="360div255"/>     <!-- actual value = stated value * 360/255, as used for GPS_STATUS.satellite_azimuth -->
```

Reply:
Or to put it another way, I have no strong opinions, but good to put this in.

Reply:
Personally, I'd prefer "360/255" because, it is both:
- directly evaluatable in pretty much any programming language for when the actual value is required for maths.
- humanly understandable as text if printed as-is (e.g. web page, Wireshark dissector, code comments, etc).

Reply:
Let's see what @peterbarker says - I dislike the slash sorry!

Reply:
I like the slash.  It means we can continue to just eval these things.

Reply:
i prefer the slash

Reply:
Great!

Reply:
Thanks for pushing this @shancock884